src/main/java/com/pro_crafting/tools/jasperreport/CompileTask.java

```diff
@@ -61,7 +60,7 @@ public Void call() throws Exception {
              InputStream in = new BufferedInputStream(new FileInputStream(source))) {
             JasperCompileManager.compileReportToStream(in, out);
             if (verbose) {
-                log.info("Compiling " + source.getName());
+                LOGGER.info("Compiling " + source.getName());
```

Comment:
Maybe use "Parameterized Logging" for all changed or new loggers?

Reply:
Thanks for the review. Yes I agree. Now that we are using slf4j, lets also switch to parameterized logging. Follow up pr coming soon.